apps/nova/src/components/TutorProfile/Ratings.tsx

```diff
@@ -1,74 +1,107 @@
+      </Typography>
+    </div>
   );
+};
 
-  const onDeleteRating = useCallback(
-    (payload: { ratingId: number; tutorId: number }) =>
-      setDeleteRating(payload),
-    []
-  );
+const Ratings: React.FC<{ id: number; tutorName: string | null }> = ({
+  id,
+  tutorName,
+}) => {
+  const { user } = useUser();
+  const intl = useFormatMessage();
+  const ratingsQuery = useFindTutorRatings(id);
 
-  const onClose = useCallback(() => {
-    setEditRating(null);
-    setDeleteRating(null);
-  }, []);
+  const refetchRatings = useCallback(() => {
+    ratingsQuery.refetch();
+  }, [ratingsQuery]);
 
-  const timeline = useMemo((): TimelineItem[] => {
-    if (!query.data) return [];
+  if (ratingsQuery.isLoading || ratingsQuery.isPending)
+    return <LoadingTutorInfo />;
 
-    return query.data.list.map((rating) => ({
-      id: rating.id,
-      children: (
-        <Rating
-          rating={rating}
-          onEdit={onEditRating}
-          onDelete={onDeleteRating}
-        />
-      ),
-      icon: <Star />,
-    }));
-  }, [onDeleteRating, onEditRating, query.data]);
+  if (ratingsQuery.error || !user)
+    return <TutorLoadingError refetchRatings={refetchRatings} />;
 
-  if (query.isLoading) return <Loading className="h-60" />;
-  if (query.isError) return <h1>Error</h1>;
-  if (!query.data) return null;
+  const ratings = ratingOrganizer(ratingsQuery.data.list, user.id);
 
   return (
-    <div>
-      <Title id="labels.ratings" />
-      <Timeline timeline={timeline} />
-      {editRating ? (
-        <EditRating open close={onClose} rate={editRating} />
-      ) : null}
-      {deleteRating ? (
-        <DeleteRating
-          tutorId={deleteRating.tutorId}
-          open
-          close={onClose}
-          id={deleteRating.ratingId}
-        />
-      ) : null}
+    <div className="flex my-8 mx-10 gap-4 flex-wrap justify-center">
+      {isEmpty(ratings) ? <NoTutorRatings tutorName={tutorName} /> : null}
+
+      {ratings.map((rating) => {
+        if (
+          "userId" in rating &&
+          (rating.feedback || rating.userId === user.id)
+        )
+          return (
+            <TutorRatingCard
+              feedback={rating.feedback}
+              imageUrl={rating.image}
+              rating={rating.value}
+              studentId={rating.userId}
+              studentName={rating.name}
+              tutorName={tutorName}
+              owner={rating.userId === user.id}
+            />
+          );
+
+        if ("ratings" in rating)
+          return (
+            <TutorRatingCardGroup
+              ratings={rating.ratings}
+              tutorName={tutorName}
+              value={rating.value}
+            />
+          );
+      })}
+
+      <div className="flex gap-10 flex-col items-center justify-center mt-10">
+        <Typography
+          element="subtitle-1"
+          weight="medium"
+          className="text-natural-950 w-2/3 text-center"
```

Comment:
Why do we have to assing `w-2/3`?

Reply:
wanted to have the sentence on 2 lines
![image](https://github.com/user-attachments/assets/3ee043ca-952d-4331-86c6-e9062e92f92e)

I will find better way. I think it's better to have it on 2 lines and not let it stretch to be easier to read

Reply:
I divided the text in `ar-eg`. I think this is a more elegant solution